package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "brave-talk-extension",
+  "version": "0.0.1",
+  "description": "Jidesha =======",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
```

Comment:
could you please add some tests for this repo?

Reply:
also, given all the dependencies, github actions should be set up to run `npm audit` on every PR.

Reply:
github action with build and audit added.

Reply:
unresolving because npm test command still needs to be added

Reply:
test now added and included in CI